backend/src/middleware/userMiddleware.js

```diff
@@ -11,6 +13,10 @@ export default {
       return result
     },
     UpdateUser: async (resolve, root, args, context, info) => {
+      const USERNAME_MIN_LENGTH = 3 // TODO move to the correct place
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T14:11:03Z" title="Wednesday, April 17th 2019, 4:11:03 pm +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,6 +13,10 @@ export default {
       return result
     },
     UpdateUser: async (resolve, root, args, context, info) => {
+      const USERNAME_MIN_LENGTH = 3 // TODO move to the correct place
```


This is sth. to test for.